app/trakt_tools.py

```diff
@@ -1,6 +1,7 @@
 import config_tools
 from urllib.parse import urlparse
 import plex_tools
+from imdb_tools import regex_first_int
```

Comment:
I think it's odd for this method to come from `imdb_tools`, instead of maybe a `misc` module.

Reply:
I mean I agree tbh but I wasn't trying to make a whole new file for one method but I'm sure there are some others we could move around. I'd prefer fewer files in general. I'd probably have four files `config_tools`, `plex_auto_collection`, `plex_tools`, and `api_tools` maybe even not have plex tools vs api tools and just have one file for all the connections to other services. In the end, this is a conversation for another day lol